### PR TITLE
fix: Respect CenterPoint when applying rotation

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by ramezgerges — APPROVED** [review]

**Review by agneszitte — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UI.Composition/Composition/Uno/TransformableObjectExtensions.cs [hunk 1]
```diff
@@ -52,7 +52,9 @@ public static Matrix4x4 GetTransform(this I3DTransformableObject transformableOb
 		var rotation = transformableObject.RotationAngle;
 		if (rotation is not 0)
 		{
+			transform *= Matrix4x4.CreateTranslation(-transformableObject.CenterPoint);
 			transform *= Matrix4x4.CreateFromAxisAngle(transformableObject.RotationAxis, rotation);
+			transform *= Matrix4x4.CreateTranslation(transformableObject.CenterPoint);
 		}
```
Comment (jeromelaban) [inline src/Uno.UI.Composition/Composition/Uno/TransformableObjectExtensions.cs:58]: @ramezgerges were you able to create a visual test for this particular behavior?

Comment (Youssef1313) [reply src/Uno.UI.Composition/Composition/Uno/TransformableObjectExtensions.cs:58]: @jeromelaban Pretty sure you meant to tag me. I will create an issue to track writing a test. This PR was to quickly unblock RefreshContainer.

Comment (jeromelaban) [reply src/Uno.UI.Composition/Composition/Uno/TransformableObjectExtensions.cs:58]: Ah yes, it was indeed for you :) Thanks!

**Pull request merged**